packages/apollo-gateway/src/index.ts

```diff
@@ -239,37 +245,50 @@ export class ApolloGateway implements GraphQLService {
     };
   }
 
-  protected async updateComposition(options?: {
-    engine?: GraphQLServiceEngineConfig;
-  }) {
+  private async updateComposition() {
     const previousSchema = this.schema;
     const previousServiceDefinitions = this.serviceDefinitions;
     const previousCompositionMetadata = this.compositionMetadata;
 
-    if (options && options.engine) {
-      if (!options.engine.graphVariant)
-        console.warn('No graph variant provided. Defaulting to `current`.');
-      this.engineConfig = options.engine;
+    let result: Await<ReturnType<Experimental_UpdateServiceDefinitions>>;
+    try {
+      this.logger.debug('Loading configuration for gateway');
+      result = await this.updateServiceDefinitions(this.config);
+    } catch (e) {
+      this.logger.warn(
+        'Error checking for schema updates. Falling back to existing schema.',
+        e,
+      );
+      return;
     }
 
-    this.logger.debug('Loading configuration for gateway');
-    const result = await this.updateServiceDefinitions(this.config);
-
-    this.logger.debug('Configuration loaded for gateway');
-
-    if (!result.isNewSchema) return;
+    if (!('serviceDefinitions' in result)) return;
```

Comment:
Alternatively, perhaps a `this.logger.trace` level log message?  From an observability standpoint, it might be beneficial to the user to know that!

```suggestion
    if (!('serviceDefinitions' in result)) {
      this.logger.trace('No changes to service definitions.');
      return;
    }
```